project/profiles/capacity4more/modules/c4m/message/c4m_message/c4m_message.features.inc

```diff
@@ -69,8 +70,9 @@ function c4m_message_default_message_type() {
           "format" : "filtered_html"
         },
         {
-          "value" : "\\u003Cp\\u003EDear @member_full_name,\\u003C\\/p\\u003E\\r\\n\\r\\n\\u003Cp\\u003EYour administration rights for the @group_node_type \\u003Ca href=\\u0022!group_dashboard_url\\u0022\\u003E@group_node_title\\u003C\\/a\\u003E have been removed.\\u003C\\/p\\u003E\\r\\n\\r\\n\\u003Cp\\u003EIf you have any questions about your new role within this  @group_node_type, please feel free to \\u003Ca href=\\u0022!event_user_message_url\\u0022\\u003Econtact me\\u003C\\/a\\u003E.\\u003C\\/p\\u003E\\r\\n\\r\\n\\u003Cp\\u003E@event_user_full_name\\u003Cbr \\/\\u003E\\r\\n@group_node_type Administrator, @group_node_title\\u003C\\/p\\u003E\\r\\n",
-          "format" : "filtered_html"
+          "value" : "\\u003Cp\\u003EDear @member_full_name,\\u003C\\/p\\u003E\\r\\n\\r\\n\\u003Cp\\u003EYour administration rights for the @group_node_type \\u003Ca href=\\u0022!group_dashboard_url\\u0022 rel=\\u0022nofollow\\u0022\\u003E@group_node_title\\u003C\\/a\\u003E have been removed.\\u003C\\/p\\u003E\\r\\n\\r\\n\\u003Cp\\u003EIf you have any questions about your new role within this @group_node_type, please feel free to \\u003Ca href=\\u0022!event_user_message_url\\u0022 rel=\\u0022nofollow\\u0022\\u003Econtact me\\u003C\\/a\\u003E.\\u003C\\/p\\u003E\\r\\n\\r\\n\\u003Cp\\u003E@event_user_full_name\\u003Cbr \\/\\u003E\\r\\n@group_node_type Administrator, @group_node_title\\u003C\\/p\\u003E\\r\\n",
```

Comment:
This is what Drupal has decided.
I made the change through UI, and then exported the feature.